packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -130,6 +136,8 @@ export async function prefetchLatestSnapshot(
 				snapshotDownloader,
 				putInCache,
 				removeEntries,
+				true, // fetchFullSnapshot
+				useGroupIdsForSnapshotFetch ? [] : undefined, // loadingGroupIds
```

Comment:
At this point, I already lost on what various combos mean, to be honest.
So fetchFullSnapshot == true & [] is default group?
And 
fetchFullSnapshot == true, and undefined full snapshot? (based on code in downloadSnapshot()).
If so Fluid.Container.FetchSnapshotUsingGetSnapshotApi means - download whole file?

Reply:
Removed "fetchFullSnapshot" and renamed FG to "Fluid.Container.UseLoadingGroupIdForSnapshotFetch".
I was ok before to have 2 fields as fetchSnapshotWithRedeem was private api. Removed that.